eps_angle: 5.0
distance_threshold: 0.01
tol_superq: 0.0001
optimizer_points: 100
random_sampling: true
max_iter: 10000000
minimum_points: 250
fraction_pc: 4
threshold_axis: 0.7
threshold_section1: 0.6
threshold_section2: 0.03
object_class: "default"
single_superq: true
merge_model: true
th_points: 100